src/pages/index.js

```diff
@@ -42,8 +42,8 @@ const api = new API({
     "Content-Type": "application/json",
   },
 });
+//test for request
 
```

Comment:
* You need to **`disable`** the submit button in the `addCardPopup` if the inputs are **empty** when you open it. It’s needed because I can click the button and add an empty card but it should **not** happen. After the first submit the button gets activated and you need to **deactivate** it again. Even if I can't add an empty card the button should look **disabled** (gray) anyway and with attribute `disabled`.
* Now only class `FormValidator` should **`disable/enable`** submit buttons and remove errors from the inputs (if you do it)


`editModalFormSelector`
`addCardFormSelector`
`avatarModalFormSelector`


They contain `#` (hash) in the selector but to take the validator from `formValidators` you need to remove it 


```js

    formValidators['add-card-form'].resetValidation();


    formValidators['modal-form-avatar'].resetValidation();


    formValidators['edit-modal-form'].resetValidation();

```


And then the validators will disable the buttons